### PR TITLE
feat: allow listview background colors to be set via theme

### PR DESCRIPTION
**Pull request by GroovinChip**
# Submit a pull request

## CLA

- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required).
- [x] The code changes follow best practices
- [x] Code changes are tested (add some information if not applicable)

### CODE REVIEW

**Review by imtoori — CHANGES_REQUESTED** [review]
LGTM but let's add an entry in the changelog for this

**Comment by GroovinChip** [comment]
> LGTM but let's add an entry in the changelog for this

Done

File: packages/stream_chat_flutter/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+## Upcoming
+- Allow the various ListView widgets to be themed via ThemeData classes
```
Comment (imtoori) [inline packages/stream_chat_flutter/CHANGELOG.md:2]:
```suggestion

✅  Added

- Allow the various ListView widgets to be themed via ThemeData classes
```

**Comment by GroovinChip** [comment]
@imtoori Are you able to tell which test is failing?

**Review by imtoori — APPROVED** [review]
LGTM

**Pull request merged**